Apps/WebClient/src/ClientApp/app/components/timeline/comment.vue

```diff
@@ -27,9 +27,31 @@
   height: 38px !important;
 }
 
-.comment-button {
+.comment-button-wrapper {
```

Comment:
instead of using @media bootstrap comes with a set of selectors that act on different sizes. For example col-md-5 will act only if the screen is size medium. You can check the landing for more since it uses it a lot.